stack_prometheus/src/ls_telemetry.conf

```diff
@@ -18,6 +18,8 @@ input {
 		'ipslacommon', 'RootOper~IPSLA~OperationData~(?<operation>\d+)~Common',
 		'counters', 'RootOper~InfraStatistics~(?<InterfaceName>.*)~Latest~GenericCounters',
                 'datarates', 'RootOper~InfraStatistics~(?<InterfaceName>.*)~Latest~DataRate']
+	      #
+	      protofiles => "/data/proto"
```

Comment:
This might give the impression that gpb is supported with the metrics. I seem to recall we only supported metrics with JSON embedded keys.